tutorials/requirements.txt

```diff
@@ -10,5 +10,9 @@ pandas>=1.5.2
 dask[distributed]
 psutil
 s3fs
+firefly_client
+jupyter_firefly_extensions
 # For supporting myst-based notebooks
 jupytext
+# For firefly-client packaging bug
+setuptools
```

Comment:
Is this something we need to fix at package configuration of firefly-client by defining setuptools as required dependency?

Reply:
yeap, see https://github.com/Caltech-IPAC/firefly_client/issues/60, as well as the other `pkg_resources` related issue. In fact addressing Tim's would make this one irrelevant, too.

Reply:
Ah, didn't notice that - thanks. I'll get this added to Jira